test/unit/src/common/actions/t_register-name.js

```diff
@@ -80,15 +82,16 @@ describe('register name actions', () => {
   });
 
   afterEach(() => {
+    storeApi.done();
     scope.done();
     nock.cleanAll();
     localForageStub.clear();
   });
 
   context('internalRegisterName', () => {
     it('throws an error on failure to register snap name', async () => {
-      scope
-        .post('/api/store/register-name', { snap_name: 'test-snap' })
+      storeApi = nock(conf.get('STORE_API_URL'))
```

Comment:
`storeApi` seems to be already initalized as nock instance, why doing it again in every test case?